testing_app/test/widget_test.dart

```diff
@@ -1,30 +1,18 @@
-// This is a basic Flutter widget test.
-//
-// To perform an interaction with a widget in your test, use the WidgetTester
-// utility that Flutter provides. For example, you can send tap and scroll
-// gestures. You can also use WidgetTester to find child widgets in the widget
-// tree, read text, and verify that the values of widget properties are correct.
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
 
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 import 'package:testing_app/main.dart';
 
 void main() {
-  testWidgets('Counter increments smoke test', (WidgetTester tester) async {
+  testWidgets('Smoke test', (tester) async {
```

Comment:
As we are already testing all the screens separately, I'm wondering if this file has become unnecessary. What are your thoughts on this?

Reply:
If it doesn't add to the narrative, it gets cut. 

On the flipside, demonstrating how to test individual widgets standalone outside of the screens they are embedded in sounds like a useful demo...

Reply:
Either add a TODO or delete outright for the time being

Reply:
I've deleted it for now. I think we can do it in the month of August. I've noted it down.

Reply:
sgtm